testkit/src/main/java/org/apache/calcite/sql/parser/SqlParserTest.java

```diff
@@ -5102,6 +5102,12 @@ void checkPeriodPredicate(Checker checker) {
             + "WHERE (`EMPNO` = 12)");
   }
 
+  @Test void testUpdateTableAlias() {
```

Comment:
Add jira link

Reply:
I added a comment with the link to the Jira issue